Traceback (most recent call last):
  File "gesture_data/server.py", line 115, in <module>
    sys.exit( main(sys.argv) )
  File "gesture_data/server.py", line 101, in main
    httpd = serverClass( (HOST_NAME, PORT_NUMBER), GestureHandler )
  File "/usr/lib/python2.7/SocketServer.py", line 419, in __init__
    self.server_bind()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/lib/python2.7/SocketServer.py", line 430, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
